app/src/main/res/layout/download_dialog.xml

```diff
@@ -106,4 +107,16 @@
             android:max="31"
             android:progress="3" />
     </LinearLayout>
+
+    <org.schabi.newpipe.views.NewPipeTextView
```

Comment:
You're not using this anymore if I understand correctly, then remove it (and also the string)

Reply:
> You're not using this anymore if I understand correctly

No, you didn't understand correctly :)

This message is always shown in the download dialog, even when there are no streams removed (the previous message was shown even when some streams were removed but at least one stream for each original resolution was still available for download).

Reply:
Oh, ok, makes sense